commerce-add-ons/payment-providers/zero/configuring-umbraco.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Configuring Umbraco
+description: >-
+  Learn how to configure the Umbraco backoffice for enabling the use of
+  Zero as a payment method.
+---
+
+# Configure Umbraco
+
+This article provides detailed instructions on configuring Umbraco to use the Zero payment method with your Umbraco Commerce implementation.
+
+## Step 1: Create a Payment Method
+
+To create Zero as a payment method, follow these steps:
+
+1. Go to the **Settings** section.
+2. Open the **Commerce** folder.
+3. Navigate to **Stores > {Store Name} > Payment Methods**.
+4. Click the **Create Payment Method** button.
+5. Select **Zero Value** from the list of available payment providers.
+
+  ![Zero Payment Provider](../media/zero/zero-payment-provider.png)
```

Comment:
When previewing on GitBook, all steps below the image are in the same line for some reason 🤔
Not sure how to resolve that.

The same applies to line 37 and the steps below that.

Reply:
I'm a bit unsure about what's happening here because the format seems to be the same as the other articles that contain steps. 🙈 Anyway, I also added a space after the image, and now the steps preview properly on GiBook as well.